src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -193,6 +193,37 @@ public function get_content_type() {
 	}
 
 	/**
+	 * Translates the posttype-driven content types to content types that the CSV importer knows
+	 *
+	 * @param string $content_type Content Type
+	 *
+	 * @return string CSV Importer compatible content type
+	 */
+	public function get_csv_content_type( $content_type = null ) {
+
+		if ( ! $content_type ) {
+			$content_type = $this->get_content_type();
+		}
+
+		switch ( $content_type ) {
```

Comment:
Hey can I suggest instead of the switch:
```php
$map = array(
     'event' => 'events'
     'events' => 'events'
     'organizer' => 'organizers'
     'organizers' => 'organizers'
     'venue' => 'venues'
     'venues' => 'venues'
);

if ( ! isset( $map[ $content_type ] ) ) {
     return false;
}

return $map[ $content_type ];
```